src/apiCalls.js

```diff
@@ -53,4 +54,11 @@ const checkForError = (response) => {
   }
 }
 
-export default { getData, postData };
+// const checkForLoginError = () => {
```

Comment:
I can't comment above, but you may need to fetch just the hotel and single customer data again after a successful booking?